core/src/avm2/globals/flash/display/NativeWindow.as

```diff
@@ -0,0 +1,317 @@
+package flash.display
+{
+  import flash.geom.Point;
+  import flash.geom.Rectangle;
+  import flash.events.NativeWindowBoundsEvent;
+  import flash.events.Event;
+  import flash.events.EventDispatcher;
+  import flash.desktop.NativeApplication;
+  import __ruffle__.stub_method;
+  import __ruffle__.stub_getter;
+  import __ruffle__.stub_setter;
+  import __ruffle__.stub_constructor;
+  
+  [API("661")]
+  public class NativeWindow extends EventDispatcher
+  {
+    public const systemMaxSize:Point = new Point(2880, 2880);
+    public const systemMinSize:Point = new Point(1, 1);
+    public var minSize:Point = systemMinSize;
+    public var maxSize:Point = systemMaxSize;
+    public var title:String;
+    public var alwaysInFront:Boolean = true;
+    public var visible:Boolean = true;
+
+    private var _bounds:Rectangle;
+    private var _maximizable:Boolean;
+    private var _minimizable:Boolean;
+    private var _resizable:Boolean;
+    private var _systemChrome:String;
+    private var _transparent:Boolean;
+    private var _type:String;
+    private var _closed:Boolean = false;
+    private var _stage:Stage;
+
+    public function NativeWindow(initOptions:NativeWindowInitOptions, _stage:Stage = null)
+    {
+      stub_constructor("flash.display.NativeWindow");
+      NativeApplication.nativeApplication._openedWindows.push(this);
+      if (_stage)
+      {
+        this._stage = _stage;
+        _stage.addEventListener(Event.RESIZE, function(e:Event):void
+        {
+          dispatchEvent(new NativeWindowBoundsEvent(NativeWindowBoundsEvent.RESIZE, false, false, _bounds, _bounds = new Rectangle(x, y, width, height)));
+        });
+      }
+      else
+      {
+        stub_constructor("flash.display.NativeWindow without stage");
```

Comment:
Minor nit: I think just (either) one of this, or the generic stub note at the top, should suffice.